libkbfs/tlf_journal.go

```diff
@@ -1902,19 +1902,19 @@ func (j *tlfJournal) getMDRange(
 }
 
 func (j *tlfJournal) doPutMD(ctx context.Context, rmd *RootMetadata,
-	mdInfo unflushedPathMDInfo,
-	perRevMap unflushedPathsPerRevMap) (
-	mdID kbfsmd.ID, retryPut bool, err error) {
+	mdInfo unflushedPathMDInfo, perRevMap unflushedPathsPerRevMap,
+	verifyingKey kbfscrypto.VerifyingKey) (
```

Comment:
I'm not sure where to put this comment, because it's percolated everywhere. But I'm confused why `verifyingKey` needs to be passed everywhere. Wouldn't it be cleaner to add `currentSessionGetter` or make a new `verifyingKeyGetter` and add it to `tlfJournalConfig`?